scripts/build-vyos-image

```diff
@@ -255,6 +255,18 @@ deb http://security.debian.org/debian-security buster/updates main non-free
 """
             fp.write(re.sub('(deb|security).debian.org', build_config['debian_mirror_server'], buster_repo))
 
+        debian_code_names = ["trixie"]
```

Comment:
The Debian distribution codename is stored in [data/defaults.toml](https://github.com/vyos/vyos-build/blob/current/data/defaults.toml#L5) and it's available in `build_config`. It must not be hardcoded anywhere in the script.

Reply:
`bookworm` is defined but `trixie` is used here.

Default code name is processed by live-build automatically, only your special list required by some specific packages are defined here.

`buster` uses a different dist names from later releases, that why I left an array for trixie and other future possible  lists.

I understand this is a bad design across debian versions. 
Another possible solution is to add another array entry in tomls, build script reads it and modify it's default debian host names.